File: php/algos/sorting/select_sort.php
<?php
function select_sort(&$arr){
	$len = count($arr);
	for($i=0;$i<$len;$i++){
		$lowest = $i;
		for($j=$i+1;$j<$len;$j++){
			if($arr[$j] < $arr[$lowest])
				$lowest = $j;
		}
		if($i !== $lowest){
			$tmp = $arr[$i];
			$arr[$i] = $arr[$lowest];
			$arr[$lowest] = $tmp;
		}
	}
}

$arr = [10,9,8,5,3,2,1,0];
select_sort($arr);
print_r($arr);

File: python/algos/sorting/select_sort.py
def select_sort(array):
	arr_len = len(array)
	for i in range(0, arr_len):
		lowest = i
		for j in range(i+1, arr_len):
			if array[j] < array[i] :
				lowest = j
		if i != lowest:
			array[i], array[lowest] = array[lowest], array[i]

arr = [3,2,1,6,9,7]
select_sort(arr)
print(arr)
